WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -605,15 +605,25 @@ private extension AuthenticationManager {
             storePickerCoordinator?.start()
         }
 
-        // Start the store creation process if the user
-        // logged in from the store creation flow.
+        // If the user is from the store creation flow
```

Comment:
super nit: I think it's nice keeping the "login" detail in the comment like below, since the user can also signs up from the store creation flow
```suggestion
        // If the user logs in from the store creation flow
```

Reply:
Done in 68e14ce29893fc59e2014a95b42c1a57f3b65544